### DISCUSSION TITLE
I am stuck, what is "sideloading" and what to do?

### DISCUSSION THREAD

**Discussion by becasud** [microsoft/hack-together-teams#58] [Q&A]
![WhatsApp Image 2023-06-11 at 9 45 30 PM](https://github.com/microsoft/hack-together-teams/assets/58492649/bd12deca-8999-41d0-abfb-8ee0054661a8)

I tried connecting my visual studio with Microsoft 365 and that's what shows for me. The debugging part does not work either because of that.

**Comment by waldekmastykarz** [comment]
Have you clicked on the "Learn More" button? It will take you to the page that explains it more detail. To explain it briefly: it's a setting that allows you to upload apps to Microsoft Teams directly, without publishing them in the app catalog.

Are you using the M365 from your organization? If so, it's possible that sideloading is disabled for governance/compliance reasons. In such case, you could request a developer tenant at https://developer.microsoft.com/microsoft-365/dev-program. You'll get a tenant where you'll have administrative privileges and will be able to control all settings including sideloading.

**Reply by becasud** [reply]
Yes, I am using M365 from my organization. I tried reading through the "learn more page", it didn't help me.

I will try to request the developer tenant then!

**Reply by waldekmastykarz** [reply]
It could be indeed that your organization is not allowing it. The best way forward would be for you to use the M365 developer tenant indeed.
